packages/contracts/src/plugins/governance/admin/build-metadata.json

```diff
@@ -1,9 +1,21 @@
 {
   "ui": "",
-  "change": "",
+  "change": "This is the first build of the first release.",
   "pluginSetupABI": {
```

Comment:
```suggestion
  "description": "A plugin that allows a certain address to execute proposals on the target DAO immediately.",
  "pluginSetupABI": {
```

Reply:
Shouldn't this general description be on the release metadata?